packages/shorebird_cli/lib/src/commands/release/ios_releaser.dart

```diff
@@ -82,8 +81,9 @@ To change the version of this release, change your app's version in your pubspec
 
     final flutterVersionArg = argResults['flutter-version'] as String?;
     if (flutterVersionArg != null) {
-      if (Version.parse(flutterVersionArg) <
-          minimumSupportedIosFlutterVersion) {
+      final version =
```

Comment:
is it okay for version to be null here?

Reply:
It is, although this comment let to me uncovering a different bug, so thank you :)

If this returns null, the command fails as so:

```
⑆ shorebird release ios --flutter-version=asdfasdfasdf
✓ Fetching apps (0.2s)
Version asdfasdfasdf not found. Please open an issue to request a new version.
Use `shorebird flutter versions list` to list available versions.


If you aren't sure why this command failed, re-run with the --verbose flag to see more information.

You can also file an issue if you think this is a bug. Please include the following log file in your report:
/Users/bryanoltman/Library/Application Support/shorebird/logs/1725467364282_shorebird.log
```